resource/tasks.json

```diff
@@ -6512,6 +6512,34 @@
         ],
         "templThreshold": 0.95
     },
+    "InfrastTraining": {
+        "template": "Training.png",
+        "maskRange": [
+            1,
+            255
+        ],
+        "roi": [
+            880,
+            420,
+            400,
+            160
+        ],
+        "templThreshold": 0.95
```

Comment:
Do we really need a threshold this high?

Reply:
The threshold of other facilities is also so high; I wrote it following their example. qwq